src/samarium/template.py

```diff
@@ -3,7 +3,7 @@
 
 STDOUT = sys.stdout
 sys.stdout = open(os.devnull, "w")
-{{ CODE }}
+{{CODE}}
```

Comment:
This breaks the template
```suggestion
{{ CODE }}
```